src/test/java/bio/terra/fixtures/FlightStates.java

```diff
@@ -0,0 +1,78 @@
+package bio.terra.fixtures;
+
+import bio.terra.model.StudySummaryModel;
+import bio.terra.service.JobMapKeys;
+import bio.terra.stairway.FlightMap;
+import bio.terra.stairway.FlightState;
+import bio.terra.stairway.FlightStatus;
+import org.springframework.http.HttpStatus;
+
+import java.sql.Timestamp;
+import java.text.SimpleDateFormat;
+import java.time.Instant;
+import java.util.Optional;
+
+
+public final class FlightStates {
+    private FlightStates() {}
+
+    public static final String testFlightId = "test-flight-id";
+    public static final Timestamp submittedTime = Timestamp.from(Instant.now());
+    public static final String submittedTimeFormatted = new SimpleDateFormat("yyyy-MM-dd'T'HH:mm:ss.SSS")
```

Comment:
is this supposed to be testing the date format? if not, should you use
    @Autowired
    private SimpleDateFormat modelDateFormat;